### DISCUSSION TITLE
UI reset

### DISCUSSION THREAD

**Discussion by gabrielmission** [TabularEditor/TabularEditor3#960] [🙋♀️TE3 Q&A] [answered]
I'm not sure what happened but all the toolbars are gone in the UI. Is there a way to do a reset back to the original setting?

![image](https://github.com/TabularEditor/TabularEditor3/assets/140655468/9f19e415-beda-49e2-ae33-86bdb5165abc)

**Comment by otykier** [comment, marked as answer]
Yes, delete the Layouts.json file within your %LocalAppData%\TabularEditor3 folder.
